### DISCUSSION TITLE
How to view the differences between the initial version and the latest version of a file on GitHub's web page

### DISCUSSION THREAD

**Discussion by AtsushiSakai** [community/community#51725] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Please teach me how to view the differences between the initial version and the latest version of a file on GitHub's web page.

**Comment by RyderCRD** [comment]
I think the _**Blame**_ button might be helpful:

1. Find the code file page, and click on _**Blame**_.

![image](https://user-images.githubusercontent.com/39330401/229819340-00d7ef3b-e8a0-4a00-9806-74f7639d1e28.png)
<br>


2. Then it will be easier to see the history of each section.

![image](https://user-images.githubusercontent.com/39330401/229819905-6767dca6-1bf0-4e2f-b6bb-c4cb6ed1db70.png)
<br>

3. And you can click on this to see the previous version, although this doesn't directly compare them to show the differences between the two.

![image](https://user-images.githubusercontent.com/39330401/229822727-28325325-881a-4b70-9836-7ef9bf898165.png)
<br>

Here are the corresponding [GitHub Docs](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#viewing-the-line-by-line-revision-history-for-a-file).

I don't think GitHub currently seems to have the ability to directly compare the initial version to the latest version.

So to directly compare the initial version and the latest version, you may need to copy them to [diffchecker](https://www.diffchecker.com/).

![image](https://user-images.githubusercontent.com/39330401/229827319-7ba9d28a-8d6b-4888-9377-7320c18a7df2.png)
